test/eventlistener_scale_test.go

```diff
@@ -138,4 +138,35 @@ func createServiceAccount(t *testing.T, c *clients, namespace, name string) {
 		t.Fatalf("Error creating RoleBinding: %s", err)
 	}
 
+	_, err = c.KubeClient.RbacV1().ClusterRoles().Create(context.Background(),
+		&rbacv1.ClusterRole{
```

Comment:
Since we are creating cluster-scoped resources, we should also delete them in a cleanup function. (By default we delete the namespace which deletes all the namespaced resources but not the cluster ones).

(e.g. we have  a cleanup https://github.com/tektoncd/triggers/blob/master/test/eventlistener_test.go#L608 that is then called in the test: https://github.com/tektoncd/triggers/blob/master/test/eventlistener_test.go#L127)

Reply:
Got it. Besides, I'm curious about https://github.com/tektoncd/triggers/blob/master/test/eventlistener_test.go#L608. I would like to know why we need to delete eventlistener after the whole namespace is already deleted. It seems that the eventlistener lives along with the namespace.

Reply:
hmm, that's a good question 🤔 

We probably should just delete the EL first and then the namespace? Or just the namespace.

Reply:
So, if TEST_KEEP_NAMESPACES env var is set, we will not delete the namespace but will delete the EL and the servcie . I wonder if this is intentional behavior.

Reply:
Thanks for the explanation. It makes sense. Should I also delete the eventlistener in test/eventlistener_scale_test.go after tear down the namespace? I can open another PR.

Reply:
Actually, maybe we can just remove the lines that delete the EventListener and instead rely on the cleanup that already deletes the namespace to clean it up?